common/common-framework/src/test/java/cn/iocoder/common/framework/util/DateUtilTest.java

```diff
@@ -0,0 +1,56 @@
+package cn.iocoder.common.framework.util;
+
+import org.junit.Test;
+import java.util.Date;
+import java.util.GregorianCalendar;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNull;
+import static org.junit.Assert.assertTrue;
```

Comment:
Don't import the static `Asserts`.

Reply:
@br As mentioned on another PR, have we received feedback that this type of static import is undesirable? I know that wildcards are frowned upon but was under the impression that static imports were acceptable.

Reply:
I think this one is more of a personal preference for me now, as I had to change a few from static wildcards to non static imports. :)

Also I think you mentioned someone else `@br` :)

Reply:
Thanks! Changed the mention to you @Braavos96 :-)